doc/usage.rst

```diff
@@ -160,6 +160,14 @@ You can close this box by clicking on the *Close* button:
 
 .. figure:: img/screenshot_job_close_details.*
 
+The Jobs view now takes Slurm's Private Data parameter into account. This feature
+is built to be paired with the new output from the update of private data
+parameter in edf-hpc/`pyslurm <https://github.com/edf-hpc/pyslurm/>`_. The original
+description can be found in`Slurm <https://slurm.schedmd.com/slurm.conf.html>`_.
+If ``jobs`` has been defined previously in Slurm configuration, regular users can
+only see their own jobs whereas the superusers see all the jobs and guests can see
```

Comment:
Slurm-web has no notion of _superusers_. There are _admins_, _users_ and _guests_ but none _superusers_.